Library/Homebrew/dev-cmd/pull.rb

```diff
@@ -111,7 +112,7 @@ def pull
     ARGV.named.each do |arg|
       arg = "#{CoreTap.instance.default_remote}/pull/#{arg}" if arg.to_i.positive?
       if (testing_match = arg.match %r{/job/Homebrew.*Testing/(\d+)/})
-        tap = ARGV.value("tap")
+        tap = @args[:tap]
```

Comment:
@sjackman Flag values can be accessed using `dot` operator, so `@args.tap` would also work.

This is my bad. I missed declaring `--tap` here https://github.com/Homebrew/brew/pull/3983 as it was not listed in man page on `brew pull`

Reply:
I believe `@args.tap` did not work, though I don't recall the error message just now. I believe @alyssais 's PR https://github.com/Homebrew/brew/pull/4003 addresses that issue.

Reply:
The error message was: `Error: no block given`